tsc/CMakeLists.txt

```diff
@@ -21,6 +21,9 @@ project(TSC C CXX)
 set(CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake/modules")
 include(ExternalProject)
 include(GNUInstallDirs)
+if(WIN32)
+  set(CMAKE_BUILD_TYPE "Release")
```

Comment:
Please don't do this. There might well be reasons to have a debug build on Windows, and the debug/release build options should behave identical on all operating systems.

Reply:
Actually, that's just a leftover. I think the executable should be stripped before being packaged, but it doesn't. I added this to see if it'll help; it did not.